VERSION

```diff
@@ -1 +1 @@
-0.0.15
\ No newline at end of file
+0.0.16
```

Comment:
I'm a bit puzzled why this is showing up as a change. Could you try rebasing instead of merging? Merging with mainline can produce strange situations with git diff.